contracts/manifest_mainnet.json

```
@@ -1252,6 +1252,8 @@
   },
   "contracts": [
     {
+      "address": "0x3d445b63c8653983b8a5b8c5c0e52f7b944f9ce3b3ecf2cb208921e400feec1",
```

Biome (1.9.4)

[error] The key address was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)